api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1114,6 +1114,27 @@ message Cluster {
   // If `connection_pool_per_downstream_connection` is true, the cluster will use a separate
   // connection pool for every downstream connection
   bool connection_pool_per_downstream_connection = 51;
+
+  // This field specifies the load balancing policy. A heterogenous mix of built-in
+  // (e.g. round robin) load balancers as well as custom implementations can be included.
+  //
+  // Built-in load balancers will have their corresponding configuration messages in
+  // typed_config, e.g. :ref:`RoundRobinLbConfig` for round robin. Custom load balancers
```

Comment:
Is there any reason why these messages could not be reused in TypedExtensionConfig?